docs_src/microservices/security/ds-proxy-entrypoint.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/dumb-init /bin/sh
+#  ----------------------------------------------------------------------------------
+#  Copyright (c) 2020 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0'
+#  ----------------------------------------------------------------------------------
+
+set -e
+
+# Use dumb-init as PID 1 in order to reap zombie processes and forward system signals to 
+# all processes in its session. This can alleviate the chance of leaking zombies, 
+# thus more graceful termination of all sub-processes if any.
+
+# runtime directory is set per user:
+XDG_RUNTIME_DIR=${XDG_RUNTIME_DIR:-/run/user/$(echo $(id -u))}
+export XDG_RUNTIME_DIR
+
+# debug output:
+echo XDG_RUNTIME_DIR $XDG_RUNTIME_DIR
+
+# use static ssh key
+rm -rf /root/.ssh && mkdir /root/.ssh \
+&& cp -R /root/ssh/* /root/.ssh/ \
+&& chmod -R 600 /root/.ssh/* \
+&& ls -al /root/.ssh/* \
+&& cat /root/.ssh/id_rsa.pub
+
+posthook="ssh -vv -o StrictHostKeyChecking=no -N $TUNNEL_HOST \
+  -L *:$LOCAL_PORT:$REMOTE_HOST:$REMOTE_PORT \
+  -R 48080:$LOCAL_HOST:48080 \
+  -R 48082:$LOCAL_HOST:48082 \
+  -R 5563:$LOCAL_HOST:5563 \
```

Comment:
I am thinking there should be an env var used for each service, in particular, is it possible to use Clients_CoreData_Host, et al here?